### ISSUE TITLE
performace gap

### ISSUE THREAD

**Issue by asdfqwer2015** [xingyul/Sparse-Winograd-CNN#6]
Hi:
  Thx for sharing your code. 

  En, I tried to reproduct the result of cifar10 in the paper with VGG. And I used the initializers same with initializers in imagenet-resnet code.  Could you please point what I missed?

  But the eval results for conventional conv(93.3+%)/ conventional winograd/ winograd-relu have obvious gaps(0.6%~1.2% for the gap between conventional conv and conventional winograd and the gap between conventional winograd and winograd-relu). I've tried a. both train with the same hyperparam or b. the better hyperparam for each model.
  
  And the similar gaps(0.5%~1.2%) can be found in resnet18 for cifar10(conventional conv with val acc 94.8+%).

**Comment by xingyul** [comment]
Thanks for sending this note. Could you check if your VGG model is correctly implemented? Note it's not traditional VGGNet, but VGG-nagadomi. 

Also, for CIFAR experiments, the learning rate starts at 0.1 and drops 10 when loss plateaus, different from ImageNet experiments.
